src/main/tv/codely/scala_http_api/Routes.scala

```diff
@@ -23,13 +25,42 @@ object Routes extends SprayJsonSupport with DefaultJsonProtocol {
       id = "3dfb19ee-260b-420a-b08c-ed58a7a07aee",
       title = "🎥 Scala FTW vol. 1",
       duration = 1.minute,
-      category = "Screencast"
+      category = "Screencast",
+      course = Course(
```

Comment:
We should avoid embedding the `Course` model inside the `Video` one. This would be the root of a tight coupling between these entities.

Instead, we should reference it by its `CourseId`. This way, we will be able to maintain the isolation between the `video` and `course` packages because the `video` one only would contain references (`import`s) from the `shared` package where we will store the `CourseId` Value Object.

This change allow us to recover `Video` instances from the database without having to load in memory all the information regarding its `Course`, `Tags`, `Comments`, or whatever information it could contain in a future.